_posts/2020-10-28-changelog-134.markdown

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #134"
+date: 2020-10-28 12:00:00 +0300
+---
+
+
+## New Features
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6267 -->
+* Support debugging in IDEA Ultimate and RubyMine on Windows.
+  Only MSVC toolchain is supported
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6242 -->
+* Show project loading steps in the Sync tool window
+
+    <img src="/assets/posts/changelog-134/sync-tool-window.png" alt="Sync tool window" width="740px"/>
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5646 -->
+* Provide completion for lint attributes (by [@Kobzol])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5871 -->
+* Check if struct fields are not bound multiple times in struct pats (see [E0025](https://doc.rust-lang.org/error-index.html#E0025)) (by [@Kobzol])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6107 -->
+* Move refactoring improvements:
+    * Support move multiple files/directories
+    * Support move a file to another crate
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6205 -->
+* Clear variables view on `:clear` command in REPL
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6204 -->
+* Support `Go to Declaration` for Cargo features in `Cargo.toml`
+
+## Fixes
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6279 -->
+* Take into account workspace features in `cargo check` / `clippy` integration
```

Comment:
Let's add a reference to external linter setting?

Reply:
Maybe replace "`cargo check` / `clippy` integration" with "external linter" since we use such term in plugin UI?